tests/test_image.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+IMAGE_PATH="Futura.png"
+OUTPUT_IMAGE_PATH="output_image.png"
+CANDID_LOG_FILE="test_image_candid.log"
+ERROR_LOG_FILE="test_image_error.log"
+CANISTER_ID="a4tbr-q4aaa-aaaaa-qaafq-cai"
```

Comment:
canister id should be fetched from .env:
```
source ../.env
echo "canister: $CANISTER_ID_FUTURA_BACKEND"

IMAGE_PATH="Futura.png"
OUTPUT_IMAGE_PATH="output_image.png"
CANDID_LOG_FILE="test_image_candid.log"
ERROR_LOG_FILE="test_image_error.log"
CANISTER_ID=$CANISTER_ID_FUTURA_BACKEND
```

Reply:
Ok!